src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -108,9 +112,11 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
     return namespaces;
   }
 
-  getSubscribeApis() {
+  async getSubscribeApis() {
+    await this.contextReady;
+
     // if user has given static list of namespaces let's not start watches because watch adds stuff that's not wanted
-    if (this.context?.cluster.accessibleNamespaces.length > 0) {
+    if (this.context.cluster.accessibleNamespaces.length > 0) {
```

Comment:
For this to be actually correct we need to always wait for the context to be ready. Otherwise we would return an API if this function was called before the context was ready.